server/src/lib.rs

```diff
@@ -171,20 +171,20 @@ impl CubebServer {
                 ClientMessage::StreamDestroyed
             }
 
-            ServerMessage::StreamStart(stm_tok) => {
-                let _ = self.streams[stm_tok].start();
```

Comment:
Dangerous pattern with `Result` I guess!

Reply:
Yeah. And all the tests pass. :-(